examples/chip-tool/commands/pairing/PairingCommand.cpp

```diff
@@ -35,6 +39,18 @@ CHIP_ERROR PairingCommand::Run()
     GetExecContext()->commissioner->RegisterDeviceAddressUpdateDelegate(this);
     GetExecContext()->commissioner->RegisterPairingDelegate(this);
 
+#if CONFIG_PAIR_WITH_RANDOM_ID
+    // Generate a random remote id so we don't end up reusing the same node id
+    // for different nodes.
+    srandom(static_cast<unsigned>(System::Clock::GetMonotonicMicroseconds()));
```

Comment:
do we need the srandom?

Reply:
Yes.  Otherwise `random()` self-seeds with 1 (at least on Mac), and I get the same node id every time.